ember.md

```diff
@@ -357,3 +359,20 @@ Create a build definition for the app that will:
 
 ### 11. Checklist Violations
 `npm test` (in the build definition) will catch errors and reject build
+
+
+
+<a name="addons"></a>
+## Addons
+
+<a name="versioning"></a>
+### 9.1 Versioning
+
+1. We define `1.0.0` release as the first version ready to be used by the end user. It may not be feature complete or bug free, but it should bring some benefit to our customers. As soon as the package is ready to be used in production, it needs to be on version `1.0.0`.
```

Comment:
I think it SHOULD be bug free according to the developer. It might not have been tested in PROD yet, but the dev has signed off on it having and passing tests and not having known defects.